specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/StaticSites.json

```diff
@@ -2730,6 +2851,14 @@
           "$ref": "#/definitions/StaticSiteTemplateOptions",
           "description": "Template options for generating a new repository."
         },
+        "privateEndpointConnections": {
```

Comment:
>privateEndpointConnections [](start = 9, length = 26)

embedding resources within another resource is not recommended, however I'm guessing this is the pattern/requirement from NRP team?

Reply:
Yes exactly